lib/vdc/DerivedVar.cpp

```diff
@@ -1615,6 +1615,113 @@ bool DerivedCoordVar_TimeInSeconds::VariableExists(
 }
 
 
+////////////////////////////////////////////////////////////////////////////// 
+//
+//	DerivedCoordVar_Time
+//
+////////////////////////////////////////////////////////////////////////////// 
+
+DerivedCoordVar_Time::DerivedCoordVar_Time(
+	string derivedVarName, string dimName, size_t n
+) : DerivedCoordVar(derivedVarName) {
+
+	_times.clear();
+	for (size_t i=0; i<n; i++) _times.push_back((float) i);
```

Comment:
Isn't this just an array of 0..n where a[i] = i? Its not a big deal but I just wanted to make sure this is intentional.

Reply:
Correct.